### ISSUE TITLE
Unexpected token: punc (.) at index.js

### ISSUE THREAD

**Issue by nikoloza** [lezer-parser/html#7]
```/.../node_modules/@lezer/html/dist/index.es.js: Unexpected token: punc (.)```

parcel is 1.x

```
    "parcel-bundler": "^1.12.4",
    "@babel/core": "^7.11.5",
```

it does not support `js?.prop`

**Comment by nikoloza** [comment]
and the file which has called `open-source.svg` was replaced with some function

**Comment by marijnh** [comment]
That should be resolved by 848b78b6ecc81432e62c6634d1c777ac1e9babf1. Not sure what you're talking about with an svg file being replaced by a function.
